lib/tasks/import.thor

```diff
@@ -52,6 +52,20 @@ class Import
       'star_reading' => StarReadingImporter::RecentImporter,
     }
 
+    ORDER = {
+      EducatorsImporter => 0,
+      CoursesSectionsImporter => 1,
+      EducatorSectionAssignmentsImporter => 2,
+      StudentsImporter => 3,
+      StudentSectionAssignmentsImporter => 4,
+      BehaviorImporter => 5,
+      AttendanceImporter => 5,
+      StudentSectionGradesImporter => 5,
+      X2AssessmentImporter => 6,
+      StarMathImporter::RecentImporter => 6,
+      StarReadingImporter::RecentImporter => 6,
+    }
```

Comment:
Why not just a list?

Reply:
Why not a json configuration that could be transitioned to be per school district?

Reply:
@kevinrobinson — I want to express the idea that some importers are app-critical and have dependencies — like educators which go before educator sections which go before students ... — while other importers are the same in terms of priority, like X2 assessments and STAR assessments. A list doesn't express that. 

@jhilde — My vote would be let's see what Fall River's data looks like before making this configurable. :D

Reply:
@alexsoble Sounds great!  Using `priority` instead of `order` might capture that.